src/main/java/org/padaiyal/utilities/aayvalar/regex/RegexUtility.java

```diff
@@ -0,0 +1,467 @@
+            RegexUtility.getAllMatches("\\\\w\\*", 0, result).size()));
+    result = result.replaceAll("\\\\w\\*",
+        generateString(randomObj.nextInt(randomValueLength + 1), wordCharactersRegex));
+
+    logger.debug(String
+        .format(I18nUtility.getString("RegexUtility.pattern.count"), "\\w+",
+            RegexUtility.getAllMatches("\\\\w\\+", 0, result).size()));
+    result = result.replaceAll("\\\\w\\+", generateString(randomValueLength, wordCharactersRegex));
+
+    logger.debug(String.format(I18nUtility.getString("RegexUtility.pattern.count"), "\\w",
+        RegexUtility.getAllMatches("\\\\w", 0, result).size()));
+    result = result.replaceAll("\\\\w", generateString(1, wordCharactersRegex));
+
+    // Case for \d in inputPattern.
+    logger.debug(String
+        .format(I18nUtility.getString("RegexUtility.pattern.count"), "\\d*",
+            RegexUtility.getAllMatches("\\\\d\\*", 0, result).size()));
+    result = result.replaceAll("\\\\d\\*",
+        generateString(randomObj.nextInt(randomValueLength + 1), digitCharactersRegex));
+
+    logger.debug(String
+        .format(I18nUtility.getString("RegexUtility.pattern.count"), "\\d+",
+            RegexUtility.getAllMatches("\\\\d\\+", 0, result).size()));
+    result = result.replaceAll("\\\\d\\+", generateString(randomValueLength, digitCharactersRegex));
+
+    logger.debug(String.format(I18nUtility.getString("RegexUtility.pattern.count"), "\\d",
+        RegexUtility.getAllMatches("\\\\d", 0, result).size()));
+    result = result.replaceAll("\\\\d", generateString(1, digitCharactersRegex));
+
+    // Case for \s in inputPattern.
+    logger.debug(String
+        .format(I18nUtility.getString("RegexUtility.pattern.count"), "\\s*",
+            RegexUtility.getAllMatches("\\\\s\\*", 0, result).size()));
+    result = result.replaceAll("\\\\s\\*",
+        String.join("", Collections.nCopies(randomObj.nextInt(randomValueLength + 1), " ")));
+
+    logger.debug(String
+        .format(I18nUtility.getString("RegexUtility.pattern.count"), "\\s+",
+            RegexUtility.getAllMatches("\\\\s\\+", 0, result).size()));
+    result = result
+        .replaceAll("\\\\s\\+", String.join("", Collections.nCopies(randomValueLength, " ")));
+
+    logger.debug(String.format(I18nUtility.getString("RegexUtility.pattern.count"), "\\s",
+        RegexUtility.getAllMatches("\\\\s", 0, result).size()));
+    result = result.replaceAll("\\\\s", " ");
+
+    return result;
+  }
+
+  /**
+   * Generate regex expression for the following numbers.
+   *
+   * @param number   The number to generate a regex based on.
+   * @param sqlRegex If true, we generate a regex that can be used with SQL queries, else it
+   *                 generates a normal regex
+   * @return The generated regexps, out of which even if one of them match a value, it is lesser
+   *     than the specified number.
+   */
+  public static String[] generateRegexExpressionsForPositiveNumbersLesserThan(int number,
```

Comment:
If the method is for `ForPositiveNumbersLesserThan`, Why do we need a Comparison object?

Reply:
I left it as new features will be added that will be using the Comparison enum.